hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnection.java

```diff
@@ -34,6 +34,16 @@
 @InterfaceAudience.Public
 public interface AsyncConnection extends Closeable {
 
+  /**
+   * Returns the clusterId of this connection.
+   */
+  String getClusterId2();
+
+  /**
+   * Returns this connection identity.
+   */
+  String getIdentity();
```

Comment:
Should we be using "scope" instead of "identity"?

Reply:
I use connScope now.